Readme.md

```diff
@@ -30,52 +30,48 @@ Many thanks to [@matthewmueller](https://github.com/matthewmueller) and [@rosshi
 Let's search on DuckDuckGo:
 
 ```js
-var Nightmare = require('nightmare');		
-var nightmare = Nightmare({ show: true });
+const Nightmare = require('nightmare');		
```

Comment:
I would probably say ES5 is a subset of ES6 since it's backwards compatible. But for this to be fully ES6, a destructuring import or es6 import would look better.